phpBB/language/en/email/mention_notify.txt

```diff
@@ -0,0 +1,19 @@
+Subject: User Tagged notification - "{TOPIC_TITLE}"
+
+Hello {USERNAME},
+
+You are receiving this notification because you have been tagged the topic "{TOPIC_TITLE}" at "{SITENAME}" <!-- IF AUTHOR_NAME != '' --> by {AUTHOR_NAME}<!-- ENDIF --> . No more notifications will be sent until you visit the topic.
+
+If you want to view the newest post made since your last visit, click the following link:
```

Comment:
This should link to the post with the mention, not to the newest post, so update the text + link